### ISSUE TITLE
`priority` not returned in output of `list DOMAIN MX`

### ISSUE THREAD

**Issue by eengstrom** [AnalogJ/lexicon#743]
It seems like an obvious thing to get back, but when I try `lexicon PROVIDER list DOMAIN MX`, I don't get a column or any way to see the `priority` of that record. It's not just MX either, but that's the most obvious.

Tell me if I'm missing some magic option, but I've looked and don't see it.

FWIW:

```
% lexicon --version
lexicon 3.5.3
```

**Comment by adferrand** [comment]
Yes, Lexicon has been designed primarily to help with `TXT` records manipulation. We still need to work on a new contract to support an efficient display for all kind for DNS records.
